pyproject.toml

```diff
@@ -101,6 +101,7 @@ types-aiofiles = "^0.8.3"
 sanic-testing = "^0.8"
 fastapi = {version = ">=0.65.0", optional = false}
 pytest-xprocess = "^0.18.1"
+MarkupSafe = "2.0.1"
```

Comment:
I had to pin this for until we update flask to latest, markupsafe 2.1.0 has some breaking changes and flask 1.0 requires jinja <3 which isn't being maintaned anymore: https://github.com/pallets/jinja/issues/1585